### DISCUSSION TITLE
Stable release date?

### DISCUSSION THREAD

**Discussion by 3ddelano** [Abdera7mane/cordot#2] [General]
Firstly, really amazing job on the static typing. Really high quality code. I was hoping if there is any date for stable release.

**Comment by Abdera7mane** [comment]
That is something I'm not sure of it myself, but I estimate it to be in the next month or after it, most of the core fundamental functionalities are implemented such us event dispatch, entity cache, updating cached entities, session resumes ect... I also tinkered with Discord REST API and I think the current implementation is OK for now but it lacks rate limiting, other than that what left is adding utilities functions to simplify the use of the library.
